android/src/main/java/com/fitpassu/stripepayments/StripePaymentsModule.java

```diff
@@ -54,14 +54,15 @@ public void onActivityResult(Activity activity, int requestCode, int resultCode,
 
     @Override
     public String getName() {
-        return "StripePaymentsModule";
+        return "StripePayments";
     }
 
     @ReactMethod(isBlockingSynchronousMethod = true)
-    public void init(String publishableKey) {
+    public void init(String publishableKey, String stripeAccount) {
```

Comment:
you should have the parameter Nullable , also check If stripe account == "" or null and based on that call. the Payment Config method or an overloaded method one with just publishable key and one with both , on your index.d.ts can have optional so user knows it can be null